sklearn/utils/tests/test_parallel.py

```diff
@@ -138,6 +139,7 @@ def test_check_warnings_threading():
         assert all(w == filters for w in all_warnings)
 
 
+@pytest.mark.xfail(_IS_WASM, "Pyodide always use the sequential backend")
```

Comment:
Shouldn't we just skip in that case ?

Reply:
I wondered the same ...

I am not sure about the subtle difference between skip and xfail in this case. We are xfailing other tests for Pyodide for example because there is no floating point exceptions so I thought I would use xfail but I am happy to change it to skip.

Reply:
Alright both are fine I guess :)